sdks/go/test/build.gradle

```diff
@@ -48,13 +48,13 @@ golang {
 }
 
 task flinkValidatesRunner {
-  dependsOn ":beam-sdks-go-test:build"
-  dependsOn ":beam-runners-flink_2.11-job-server:shadowJar"
```

Comment:
Does the SDK not work on 2.11, or is that version not yet portable? I'm concerned about running on an older version of flink

Reply:
AFAIK this isn't actually a downgrade, it's just a change in the way we identify Flink runner versions (apparently with Flink versions now instead of Beam) as of [1]. @angoenka can you confirm?

[1] https://github.com/apache/beam/commit/1e7ea0da5073566c3fa26dbc1105105fbe6043ae#diff-985a464c6b477d468d4fa026191dedf5

Reply:
Yes, this is just a naming change to match flink version.